doc/matchers.md

```diff
@@ -90,6 +90,10 @@ For the extension function style, each function has an equivalent negated versio
 | `double.shouldBeLessThanOrEqual(n)` | Asserts that the double is less or equal to than the given value n |
 | `double.shouldBeGreaterThan(n)` | Asserts that the double is greater than the given value n |
 | `double.shouldBeGreaterThanOrEqual(n)` | Asserts that the double is greater than or equal to the given value n |
+| `double.shouldBePositive()` | Asserts that the double is positive |
+| `double.shouldBeNotPositive()` | Asserts that the double is not positive|
```

Comment:
The documentation here is wrong. I believe you're aiming for `double.shouldNotBePositive`